pandas/core/indexes/period.py

```diff
@@ -802,6 +802,9 @@ def join(self, other, how='left', level=None, return_indexers=False,
             return self._apply_meta(result), lidx, ridx
         return self._apply_meta(result)
 
+    def intersection(self, other, sort=False):
```

Comment:
you might be able to do with a Substitution doc-string on a shared doc

Reply:
I think you can do a `@Appender(Index.intersection.__doc__)`